docs/src/pages/components/icons/icons.md

```diff
@@ -97,7 +97,7 @@ If you need a custom SVG icon (not available in the [Material Icons](/components
 This component extends the native `<svg>` element:
 
 - It comes with built-in accessibility.
-- SVG elements should be scaled for a 24x24px viewport so that the resulting icon can be used as is, or included as a child for other MUI components that use icons. (This can be customized with the `viewBox` attribute).
+- SVG elements should be scaled for a 24x24px viewport so that the resulting icon can be used as is, or included as a child for other MUI components that use icons. (This can be customized with the `viewBox` attribute, to inherit `viewBox` value from original image `inheritViewBox` attribute can be used).
```

Comment:
At this point, the parenthesis probably is not relevant.

```suggestion
- SVG elements should be scaled for a 24x24px viewport so that the resulting icon can be used as-is, or included as a child for other MUI components that use icons. This can be customized with the `viewBox` attribute, to inherit `viewBox` value from the original image `inheritViewBox` attribute can be used.
```